src/UnitTest/test_quote_prov.cpp

```diff
@@ -333,6 +359,32 @@ static HINSTANCE LoadFunctions()
 }
 #endif
 
+//
+// extract raw value from response body, if exists
+//
+sgx_plat_error_t extract_from_json(

```

Comment:
Please consider using a different name than in prod code. extract_from_nested_json ?